DEVELOPING.md

```diff
@@ -31,7 +23,7 @@ The command will produce the `revive` binary in the root of the project.
 
 If you want to develop a new rule, follow as an example the already existing rules in the [rule package](https://github.com/mgechev/revive/tree/master/rule).
 
-All rules should implement the following interface:
+Each rule needs to implement the `lint.ConfigurableRule` interface. All rules with a configuration must implement that interface:
```

Comment:
These sentences are not clear to me.
AFAIK rules must not necessary implement `Configurable` (thus _Each rule needs to implement Configurable_ is not true)

I understand "_that_" in "_All rules with a configuration must implement that interface_" as the `Configurable` interface but following the sentence there is the code of `Rule` interface (no that of `Configurable`).

Reply:
ok now should be better :)